### PR TITLE
Frontend for artist show page

### PR DESCRIPTION
**Pull request by GRSDY**
Created frontend for the artist show page

### CODE REVIEW

File: app/views/artists/show.html.erb [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
+<div class="layout-container">
+
+<div class="banner-show">
+  <h1 class="show-artist-title"><%= @artist.name%></h1>
+</div>
+
+
+<div class="artist-img-banner">
+  <% if @artist["image_url"].present? %>
+    <img id="x" src="<%= artist["image_url"] %>" alt="artist photo">
```
Comment (isaquito1993) [inline app/views/artists/show.html.erb:10]: i think here it should say @artist instead of artist

**Review by isaquito1993 — COMMENTED** [review]
the genre box seems quite big for me. there will 3-4 at most, so it doesnt make sense that its as big as the description

and maybe, but for another PR, the follow artist button could be just a heart that can be clicked and unclicked?

also, for whatever reason, the search artists is not working. I look for well known artsits but get 0 results. not sure if something with the API change? @CdLeng

**Comment by CdLeng** [comment]
@GRSDY Nice work! I noticed a bug, the @artist["image_url"]  should be @artist.image_url, because it's an hash at not an array.

@isaquito1993  yeah that's weird, because Heroku is working well. I think it has to do with the change @GRSDY made to the artist index view. You should use the instance variable @results and not @artists because the results are loaded from the Spotify API and are not in the DB yet.

Also I would place the follow button in the image header of the artist.
And the concerts should be right below the image header I think, but I'm still working on getting the concerts from the ticketmaster API

**Comment by CdLeng** [comment]
@carolinavmo can you maybe improve this?

- I think the artist image must be smaller (more square) or else it will be out of proportion
- the follow button can be on the top
- descritpion and genres can be in the same column
- concerts can be shown next to the description column

atleast that's my idea, I think you have a good feeling about how to improve it!

**Comment by carolinavmo** [comment]
Okay, I will work on it today ;)

**Comment by carolinavmo** [comment]
Some changes, to match the style of artist cards on the profile page
Screenshot 2023-03-25 at 11-37-32 Concertify

**Comment by carolinavmo** [comment]
Forgot to add the save concert button... I will do it today

**Comment by carolinavmo** [comment]
Screenshot 2023-03-25 at 15-01-59 Concertify

**Pull request merged**